bin/exec.go

```diff
@@ -85,93 +77,49 @@ func main() {
 
 		// If not found in AppNamespace skip the further steps
 		if !isFound {
+			//TODO Patch Engine if chaosExperiment is not found.
+			engineDetails.ExperimentNotFoundPatchEngine(experiment, clients)
+
 			log.Infoln("Can't Find Experiment Name : "+engineDetails.Experiments[i], "In Namespace : "+engineDetails.AppNamespace)
```

Comment:
"Cannot list chaos experiment "x" in namespace "y"

Reply:
Done